clj/README.md

```diff
@@ -0,0 +1,14 @@
+# clojure-example
+
+A Clojure library designed to ... well, that part is up to you.
+
+## Usage
+
+FIXME
```

Comment:
Mind adding quick instructions here? Should just be a "`cf push`, then visit `clojure.18f.gov`", right?